test/TestProjectWithIncludedSyntaxErrors/TestProjectWithIncludedSyntaxErrors.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="MSBuild.Sdk.SqlProj/1.16.0-beta.4">
```

Comment:
Perhaps we could setup this project to use the `Sdk.props` and `Sdk.targets` files directly as the other test projects do?